recipes/ensembl-vep/meta.yaml

```diff
@@ -9,10 +9,12 @@ source:
   sha256: 429e578e4fcfa57f7442eb564c546efb3c1f2123921a5e2ebee7ed597e657b8a
 
 build:
-  number: 0
+  number: 1
   noarch: generic
   
 requirements:
+  run_exports:
+    - perl-lwp-simple
```

Comment:
Why not just adding this to the run section?

Reply:
In addition, perl-lwp-simple seems to be one of those dummy perl packages. To me it seems like the module is already part of main perl. But I don't exactly get how this perl stuff works.

Reply:
For such purpose, run exports has to be inside of the used library though. And for perl, that is not needed anyway because this happens within the run_exports of the perl package.